### ISSUE TITLE
Sometimes (?) Pixelpulse doesn't draw on the complete screen

### ISSUE THREAD

**Issue by rgetz** [analogdevicesinc/Pixelpulse2#165]
![image](https://cloud.githubusercontent.com/assets/1017869/14208727/cd47700c-f7ee-11e5-8294-d59d1f142e2c.png)

happens on latest version.

I'm not sure if this is because something is getting confused about a device manager, or something else being on the screen, when it is really not?

Thanks

**Comment by rgetz** [comment]
To duplicate : When not using "repeated sweep", and using the "10s - sample time" option, the oscilloscope windows is never filled.

**Comment by luciansin** [comment]
I couldn't reproduce this bug on Windows 7(64bits) and PixelPulse (Version: v0.88-1-g2a3ff9b). But I've found another bug starting from the above settings. First not use "repeated sweep" and use "10s - sample time option". Afterwards if you change to "1s - sample time option", the application will be blocked.

**Comment by rgetz** [comment]
I couldn't make this happen 100% of the time. But can make it happen eventually. 

It is like one the device manage thinks it should be drawn, but isn't.

**Comment by luciansin** [comment]
You are right. It happened also to me eventually.

**Comment by jonpearson** [comment]
Could not duplicate, but probably due to not enough tries as others have noted.  This is not a high priority for the summer camp labs, but should be resolved quickly nonetheless.

**Comment by dNechita** [comment]
Fixed in commit: https://github.com/analogdevicesinc/Pixelpulse2/commit/c0d0ca2a4783659c0190510e205a7322d43ee205

I was able to always reproduce the bug if I did the following:
1.start app
2.uncheck Repeated Sweep option
3.set any channel to output a periodic voltage signal - just for a better visualization  of the issue
4. start capturing
5. set the Sample Time to a higher value than the one currently set.
